org.chromium.Chromium.metainfo.xml

```diff
@@ -1,4 +1,3 @@
-<!-- Copyright 2017 The Chromium Authors -->
```

Comment:
You can put it after the `<?xml ...?>` tag. I went to [great lengths](https://github.com/flathub/flatpak-external-data-checker/blob/8b5dfa2891b10502cc63804248634e1213a41396/src/lib/appdata.py#L92-L125) to make this work. [Here's the test](https://github.com/flathub/flatpak-external-data-checker/blob/8b5dfa2891b10502cc63804248634e1213a41396/tests/test_appdata.py#L192-L223).

Reply:
Thanks, updated - although not sure how accurate this copyright notice is.

Reply:
Agreed. But at least it's still here so it could be updated :-)